addons/vscode/package.json

```diff
@@ -77,6 +77,19 @@
         "description": "Show typst-preview log",
         "icon": "$(list-flat)",
         "when": "resourceLangId == typst"
+      },
+      {
+        "command": "typst-preview.revealDocument",
```

Comment:
Is this command visible to user?

Reply:
They are necessary, since a tree item can only bind a command action. Or we should have an approach to hide them.

Reply:
i see. I find some tricks to hide commands https://stackoverflow.com/a/55271392 hope that helps

Reply:
also this seems unneeded:

> 18

If you don't need to associate an "icon" or a "title" with your command, you can simply omit it from "commands" - commands that are not listed there can still be called, as long as they have been registered via vscode.commands.